src/navigation/RootNavigator.tsx

```diff
@@ -75,22 +78,186 @@ export default function RootNavigator() {
             <Stack.Screen
               name="TabsStack"
               component={TabsStack}
-              options={{ headerShown: false }}
+              options={({ navigation }) => ({
+                headerTitle: () => {
+                  return <LogoTitle />;
+                },
+                headerTitleAlign: 'center',
+                headerStyle: {
+                  backgroundColor: '#CFD3D7',
```

Comment:
[TO DO]: add this color to themes and import it instead! For instructions of how to do this: see [team documentation](https://www.notion.so/calblueprint/SIREN-React-Native-Styling-Guide-7e7592a27d4f4ffdae759e6bb326e867)

Reply:
will do!